lib/runner/CLI.js

```diff
@@ -20,6 +23,8 @@ process.on("uncaughtException", (e) => {
 // Also, don't want to force people to learn Jest
 class CLI {
     static displayError(e) {
+        LoggingErrorHelper.error(Logger, "Error using bst-test on Node: " + process.version);
```

Comment:
This code is very needlessly repetitive.
1. We should not pass the process.version everytime - the routine should just look at this
2. If the string "Error using bst-test on Node" never changes, it should be a constant and not passed every time.
If it's just one of two or three values, they should be consts
3. This should be just one line of code, not broken into two

In general, when we should strive to avoid this sort of repetition in our code.